src/op_handlers/div.rs

```diff
@@ -1,3 +1,6 @@
+
+use u256::U256;
```

Comment:
Is this import necessary? I don't see where it is used in the new changes.

Reply:
It wasn't, now It is :)